python/foglamp/services/core/api/configuration.py

```diff
@@ -85,6 +92,14 @@
     if category is None:
         raise web.HTTPNotFound(reason="No such Category found for {}".format(category_name))
 
+    for k, v in category.items():
+        if v['type'] == 'script':
+            prefix_file_name = category_name.lower() + "_" + k.lower() + "_"
```

Comment:
right now, if we have a file `dp.py` for category name `CAT #1` and config item `filterScript` then we are saving it 
as `cat #1_filterscript_dp.py` I'm not sure if we want it this way? 

May be we could have saved it in `cat #1_filterscript` directory to keep file name as is. 

In case filename does not matter, and we want the client to expect the name same way. Its okay!